recipes/libqdldl/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
```

Comment:
Not needed, part of `CMAKE_ARGS`.
```suggestion
```

Reply:
Great, I was not aware of it. Do you have a pointer on where `CMAKE_ARGS` is defined so that I can check with arguments are already there?

Reply:
I found it: https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/6290f55b73a887086b0efe0cdb32553646696a8a/recipe/activate-gcc.sh#L144 .

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/15140/commits/bc99b668f303198d3cd88ff77d2dd14b58a218f2 .